### ISSUE TITLE
Licensing your nREPL contributions under EPL

### ISSUE THREAD

**Issue by cemerick** [cemerick/nREPL#2]
For background, please see #1.

Your contributions to nREPL over the years are greatly appreciated. To better serve all of the users of nREPL, and to simplify things for those that are most impacted by the project, I am pulling development here (where the project began some 8 years ago).

Note that the license is _not_ changing from EPL, the license that has always governed nREPL contributions. However, your commits were submitted through the Clojure Contrib process, which carries additional terms. Since I am seeking to excise the Contrib/CA heritage from the project, and IANAL, I'm now asking to reaffirm your licensing of your nREPL commits under "regular" EPL terms, so they may be included here.

Once all contributors indicate their assent here, I will reconstruct the nREPL commit history to include only your work, which will drop things like minor build process changes, commits by the build.clojure.org buildbot, etc. My hope is that the result will be a place where people invested in nREPL and the tools that depend upon it can collaborate using familiar practices and a set of enthusiastically shared expectations.

## tl;dr

If you agree to license your contributions to [tools.nrepl](https://github.com/clojure/tools.nrepl/) under the [Eclipse Public License](http://www.eclipse.org/legal/epl-v10.html), please respond here with "OK" or similar. This will unambiguously allow those contributions to be used in other EPL-licensed contexts (including this project) without consideration of the Clojure Contribution Agreement (CA), under which those contributions were made originally.

If you do not agree, please indicate this in a comment as well, or contact me directly by [email](mailto:chas@cemerick.com).

The roster of contributors with material commits in the [tools.nrepl](https://github.com/clojure/tools.nrepl/) history are as follows:

* [x] @cemerick
* [x] @kotarak
* [x] @trptcolin 
* [x] @bbatsov
* [x] @seancorfield 
* [x] @gfredericks
* [x] @cichli
* [x] @andrewhr
* [x] @hugoduncan
* [x] @jeffvalk
* [x] @jafingerhut
* [x] @brandonbloom
* [x] @duelinmarkers
* [x] @misfo
* [x] @ato
* [x] @lprefontaine

Thank you again for your work.   ♥️

**Comment by cemerick** [comment]
OK

**Comment by bbatsov** [comment]
OK

**Comment by seancorfield** [comment]
I’m shocked to hear that I had a “material commit” to that repo but, yes, I’m perfectly happy to have whatever contribution I made to tools.nrepl licensed under the vanilla EPL.

Sean Corfield -- (970) FOR-SEAN -- (904) 302-SEAN
An Architect's View -- http://corfield.org/

"If you're not annoying somebody, you're not really alive."
-- Margaret Atwood

**Comment by jeffvalk** [comment]
OK

**Comment by trptcolin** [comment]
OK

**Comment by lprefontaine** [comment]
OK

**Comment by duelinmarkers** [comment]
OK

**Comment by andrewhr** [comment]
OK

**Comment by jafingerhut** [comment]
OK

**Comment by gfredericks** [comment]
OK

Gary Fredericks
(803)-295-0195
fredericksgary@gmail.com
gfredericks.com

**Comment by brandonbloom** [comment]
OK

**Comment by ato** [comment]
OK

**Comment by cemerick** [comment]
@seancorfield oh, you didn't need to edit. It was good! :-D

**Comment by kotarak** [comment]
OK

> Am 09.10.2017 um 20:52 schrieb Chas Emerick <notifications@github.com>:
> 
> m

**Comment by misfo** [comment]
OK

**Comment by hugoduncan** [comment]
OK

**Comment by cichli** [comment]
OK

**Comment by cemerick** [comment]
We now have unanimous consent on this issue from all tools.nrepl contributors. Thank you very much to everyone for taking the time to consider this. I'll close and lock this now, so as to keep your notifications as clean as possible; further action will happen over on #1 and related issues. 💯
